yt_dlp/extractor/stv.py

```diff
@@ -73,11 +73,12 @@ def _real_extract(self, url):
             })
 
         programme = result.get('programme') or {}
+        account_id = '6204867266001' if programme.get('drmEnabled') else '1486976045'
```

Comment:
why not just call `self.report_drm` if `drmEnabled` is true? there is no need for the drm account id to be added.

Reply:
It's an unnecessary assumption that `drmEnabled` means the video is DRM protected. The STV code just says if `drmEnabled = 1`, it's a different account ID.

Reply:
Do you have an example where `drmEnabled  = 1` but the brightcove video is not DRMed?

Reply:
No. Nor can I say that every video where `drmEnabled  = 1` is protected.

Reply:
> No. Nor can I say that every video where `drmEnabled = 1` is protected.

isn't that the implication of `drmEnabled`?

Reply:
I only want this change to gamble on that implication being false. Or on the possibility of Brightcove DRM being cracked one day. Improving the UX doesn't help me, nor the person who submitted the issue, as the videos still can't be downloaded.